rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,601 @@
+a format that groups and indexes a potentially very large number of two
+dimensional images, forming 5-dimensional tensors over (time, channel,
+z, y, x). These dimensions are fairly consistent across imaging domains
+and will likely generalize to other use cases. SpaceTx created this
+format because they were unable to find an existing format that
+satisfied our cloud compute and data visualization use cases. *starfish*
+and OME have since become aware of contemporaneous development of
+projects that aim to satisfy similar use cases and are working towards
+unification, where possible across scientific disciplines and languages.
+
+To avoid writing a large amount of microscope-specific code at the
+outset, *starfish* requests that users carry out basic pre-processing of
+their data to fix registration or chromatic errors. More details on the
+specific requests made of users can be found
+[<span class="underline">here</span>](https://drive.google.com/a/chanzuckerberg.com/open?id=1x516WMMWkEKKQwLrqqBsKmsPEDNjFYugqxm400D2NjY).The
+exact specification of the image data format will depend upon the
+delivery date that the DCP requests for this prototype.
+
+##### Pipeline Recipes
+
+The parameterization of an imaging pipeline depends on the assay in
+question, the microscope used to capture the data, the density of spots
+in the tissue, and the tissue being analyzed. For example, if the same
+assay is run a second time at twice the magnification, spots would be
+expected to have twice the radius, which would need to be communicated
+to the spot-finding algorithm.
+
+Human analysts are very good at selecting these parameters, but
+processes to fit parameters to all datasets across assays, tissues, and
+microscopes have not yet been developed. To solve this problem, starfish
+exposes the ability for users to pilot starfish on small subsets of
+their data in a local environment. These analyses generate parameter
+sets ("pipeline recipes") which are stored in json and validated by
+*starfish*. These parameters are then used to run *starfish* across the
+complete dataset, in a cloud-based environment.
+
+#### Output Data Formats
+
+Image-based transcriptomics experiments emit three output formats that
+contain identified features, all of which can be emitted in either
+netcdf or zarr file formats, the latter being the format consumed by the
+matrix service. In addition, it has the opportunity to emit image files
+that have been processed to align images, remove autofluorescence and
+enhance spots. All output formats (including SpaceTx image format)
+contain full provenance records of the analysis environment, versions of
+starfish and key dependencies, algorithms that were run, and their
+parameters.
+
+##### [<span class="underline">IntensityTable</span>](https://spacetx-starfish.readthedocs.io/en/ajc-output-specifications/sptx-format/output_formats/IntensityTable/index.html)
```

Comment:
These title links are broken(?)

Reply:
Thanks, @mweiden. That branch got merged. Links updated.